### PR TITLE
[TS-4423] Update Show Location Options

### PR DESCRIPTION
**Pull request by ericcarlschwartz**

### CODE REVIEW

**Comment by jpeach** [comment]
Please rebase to remove the merge commit.

File: cmd/traffic_ctl/traffic_ctl.cc [hunk 1]
```diff
@@ -240,7 +240,7 @@ main(int argc, const char **argv)
   if (debug) {
     diags->activate_taglist("traffic_ctl", DiagsTagType_Debug);
     diags->config.enabled[DiagsTagType_Debug] = true;
-    diags->show_location                      = true;
+    diags->show_location                      = 1;
```
Comment (jpeach) [inline cmd/traffic_ctl/traffic_ctl.cc]: Can you please add constants to ``Diags.h`` for the ``show_location`` levels?

**Comment by zwoop** [comment]
Also update the commit message :)

**Comment by ericcarlschwartz** [comment]
Dropped the merge commit and made an enum instead of magic numbers. What's wrong with the commit message @zwoop? I just grabbed the name from the bug.

**Comment by jpeach** [comment]
The preferred commit message is:
```
TS-1234: less than 80 chars.
```

Feel free to add an extended explanation into the body of the commit message. It makes it easier than having to refer back to Jira tickets and pull requests.

I think @zwoop may have been referring to the PR subject, which should generally be the same as the leading commit. The PR subject shows up as a merge commit so it is worth making it nice :)

**Comment by ericcarlschwartz** [comment]
Ah got it, will fix that up!

**Comment by zwoop** [comment]
Fwiw, Github sort of imposes a Subject length of 50 characters, it will line wrap beyond that, making for ugly messages. So, I think we should recommend

    TS-1234 < 50 character Subject
    
    Some longer text can come here.
    With more lines, and longer lines, but still < 80 characters per line.

**Comment by ericcarlschwartz** [comment]
That sounds good to me. Will try to adhere to in the future. I just went with "Update Show Location Options" for this guy

**Comment by zwoop** [comment]
Cool. And this PR still needs some fixing too on the commit message :).

**Comment by ericcarlschwartz** [comment]
Ah, my mistake there. Didn't realize it'd autopopulated w/ something bad.

**Comment by zwoop** [comment]
meh we landed this without testing on the CI :-/. It breaks the builds.

**Pull request closed without merging**